docs/sdk/configuration/fragments/ios/setup-options.md

```diff
@@ -1,5 +1,16 @@
 The AWS SDK contains [high level client interfaces](~/start/start.md) for quickly adding common features and functionality to your app. You can also manually add the generated AWS service interfaces for direct interaction if you have custom or advanced requirements.
 
+## Supported iOS platforms
+
+New versions of the AWS SDK removes support for older versions of iOS and building with Xcode. See the table below to determine which minimum iOS version you can support when using the AWS SDK.
+
+|AWS SDK version | Minimum iOS version | Xcode version |
+|------------|---------------------|---------------|
+|2.16.0 and up   | iOS 9.0 | Xcode 12 and above
```

Comment:
we had a few issues still in 2.16.0 and the SDK version for iOS 9 and Xcode12 support should really be 2.17.0 and up. perhaps we should mention 2.16.0 as a separate row here